packages/cli/tests/unit/depends.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('Depends', () => {
   beforeAll(async () => verbose(process.env.DEBUG === 'true'));
   beforeEach(async () => {
     appMapDir = tmp.dirSync().name;
-    expected = join(appMapDir, 'user_page_scenario').replace(/\\/g, '/');
+    expected = join(appMapDir, 'user_page_scenario');
```

Comment:
I assume this was so the tests pass on Windows correctly. I wonder what changed that it's not needed anymore?

Reply:
Before, we were using `glob` to enumerate the files. Now we are walking the directory tree ourselves, so the file paths don't require this gsub.